COPYING

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2003-2006 Eelco Dolstra
+Copyright (c) The NixOS and nixpkgs authors
```

Comment:
I recommend keeping the year range and updating it:

Copyright (c) 2003-2015 The NixOS and nixpkgs authors